packages/webview_flutter/webview_flutter_wkwebview/lib/src/webview_cupertino_widget.dart

```diff
@@ -0,0 +1,227 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+
+import 'package:device_info_plus/device_info_plus.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart' as material;
+import 'package:flutter/widgets.dart';
+import 'package:pub_semver/pub_semver.dart';
+import 'package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart';
+
+import 'web_kit/web_kit.dart' as web_kit;
+
+/// A [Widget] that displays a [web_kit.WebView].
+class WebViewCupertinoWidget extends StatefulWidget {
+  /// Constructs a [WebViewCupertinoWidget].
+  const WebViewCupertinoWidget({
+    required this.creationParams,
+    required this.callbacksHandler,
+    required this.javascriptChannelRegistry,
+    required this.onBuildWidget,
+    this.configuration,
+    this.systemVersion,
+    @visibleForTesting this.webViewProxy = const WebViewProxy(),
+  });
+
+  /// The initial parameters used to setup the WebView.
+  final CreationParams creationParams;
+
+  /// The handler of callbacks made made by [web_kit.NavigationDelegate].
+  final WebViewPlatformCallbacksHandler callbacksHandler;
+
+  /// Manager of named JavaScript channels and forwarding incoming messages on the correct channel.
+  final JavascriptChannelRegistry javascriptChannelRegistry;
+
+  /// A collection of properties that you use to initialize a web view.
+  ///
+  /// If null, a default configuration is used.
+  final web_kit.WebViewConfiguration? configuration;
+
+  /// The system version of the iOS device.
+  ///
+  /// If null, this value is retrieved asynchronously.
+  final FutureOr<String>? systemVersion;
+
+  /// The handler for constructing [web_kit.WebView]s.
```

Comment:
If it's for creating webviews, isn't it a `Factory` rather than a `Proxy`?

Reply:
On Android, this class is also used to call static methods. So there is a possibility that it could be used for more than constructing `WebView`s. I updated the documentation.